src/Components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,38 @@
+import "./Sidebar.css";
+import { FilterHeader } from "./FilterHeader/FilterHeader";
+import { SortFilter } from "./SortFilter/SortFilter";
+import { RatingFilter } from "./RatingFilter/RatingFilter";
+import { CartegoryFilter } from "./CategoryFilter/CategoryFilter";
+import { BrandFilter } from "./BrandFilter/BrandFilter";
+import { OtherFilter } from "./OtherFilter/OtherFilter";
+// import { useRoute } from "../../Context/useRoute";
+
+export const Sidebar = () => {
+  // const { sidebar } = useRoute();
+
+  return (
+    <aside>
+      <div
+        id="sidebar-menu"
+        className="sidebar stick position-open"
+        // className={`sidebar stick ${
+        //   sidebar ? "position-open" : "position-hidden"
+        // }`}
+      >
+        <FilterHeader />
+        <ul className="list-style-none filter-section">
+          <div className="filter-divider-line"></div>
```

Comment:
For divider line , you can use <hr />, or border-bottom property on existing div

Reply:
thanks for the suggestion